include/lemlib/MotionHandler.hpp

```diff
@@ -0,0 +1,112 @@
+#pragma once
+
+#include <functional>
+
+/**
+ * @brief this macro can be used to greatly simplify passing motion algorithms to the motion handler
+ *
+ * @b Example:
+ * @code {.cpp}
+ * void autonomous() {
+ *   MOVE_CUSTOM(simpleMotion())
+ * }
+ * @endcode
+ *
+ */
+#define MOVE_CUSTOM(f) lemlib::MotionHandler::move([&] { f; });
```

Comment:
Might be good to make the macros optional, perhaps by putting by putting them behind an `ifdef` statement. This would let the user choose whether to pollute their global namespace.

Reply:
done